src/test/benchmarks/io/vertx/benchmarks/ExecutorBenchmark.java

```diff
@@ -0,0 +1,113 @@
+package io.vertx.benchmarks;
+
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.LinkedTransferQueue;
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.infra.Blackhole;
+
+import io.netty.util.concurrent.ImmediateEventExecutor;
+
+public class ExecutorBenchmark {
```

Comment:
can we get this test to test actually Vert.x execute blocking and not plain executors ? that's the metric we are interested in :-)

Reply:
Sorry, I don't understand - Vert.x runs `Executor.execute()` method: https://github.com/eclipse-vertx/vert.x/blob/master/src/main/java/io/vertx/core/impl/ContextImpl.java#L184

Without pluggable executor factory I can't test `Vertx.executeBlocking`. Do you want me to mock all the components to excercise `BenchmarkContext.executeBlocking` with custom `WorkerPool`?

Reply:
well don' tbother just drop the benchmark for now and I'll resurect it if that's needed